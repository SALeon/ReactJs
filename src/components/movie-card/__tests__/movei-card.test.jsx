import React from 'react';
import { shallow } from 'enzyme';
import MovieCard from '../index';

const movieInfo = {
  id: '2',
  title: 'title',
  releaseDate: '2017-08',
  poster: 'https://image.tmdb.org/t/p/w500/ldoY4fTZkGISMidNw60GHoNdgP8.jpg',
  genres: ['ganre1', 'genre2'],
  voteCount: 9,
};

it('render MovieCard Component', () => {
  const component = shallow(<MovieCard movieInfo={movieInfo} />);
  expect(component).toMatchSnapshot();
});
