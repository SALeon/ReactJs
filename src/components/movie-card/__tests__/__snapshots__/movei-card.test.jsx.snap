// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render MovieCard Component 1`] = `
<div
  className="movie-card"
>
  <img
    alt="movie poster"
    className="movie-card__img"
    src="https://image.tmdb.org/t/p/w500/ldoY4fTZkGISMidNw60GHoNdgP8.jpg"
  />
  <div
    className="movie-card__info"
  >
    <span
      className="movie-card__title"
    >
      title
    </span>
    <span
      className="movie-card__release-date"
    >
      2017
    </span>
    <span
      className="movie-card__genre"
    >
      ganre1 & genre2
    </span>
  </div>
</div>
`;
